test/test-component.js

```diff
@@ -77,31 +77,20 @@ describe('Nested Component instance', function() {
   var el, childEl;
 
   context('before attached to DOM', function() {
-    beforeEach(function(done) {
+    beforeEach(function() {
       document.body.innerHTML = '';
       el = document.createElement('nested-app');
```

Comment:
well actually in order to test any of this 'before attached to DOM' context at all you need to be able to run assertions at the point when the parent template has been rendered but the child's `attachedCallback` has not yet run, which I don't think is possible with these integration-style tests. We'll have to rewrite these with the test code manipulating the components directly (e.g., manually running the created and attached callbacks)